ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__rosa-4.14-nightly.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: stream9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: prow/Dockerfile
+  from: os
+  to: krkn-hub
```

Comment:
Need to make the **to** field different in each of the config files 

Example: https://github.com/openshift/release/blob/d51689f6c022df76b52ec1d645c56ff55df2b6f7/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13.yaml#LL6C1-L8C34

https://github.com/openshift/release/blob/d51689f6c022df76b52ec1d645c56ff55df2b6f7/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main.yaml#L6-L8

https://github.com/openshift/release/blob/d51689f6c022df76b52ec1d645c56ff55df2b6f7/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml#L6-L8

Reply:
Updated. Thanks @paigerube14!